Documentation/cli-installation-scenarios.md

```diff
@@ -0,0 +1,241 @@
+
+# General principles
+
+- All installers are signed properly 
+- Upgrades using the native installers Just Work(tm)
+- All user facing materials point to the getting started page
+- The user needs extra effort to install the "bleeding edge" bits
+- Only HTTPS links are allowed in any online property 
+
+# Channels
+Channels represent a way for users who are getting the bits to reason about the stability and quality of the bits they are getting. Each channel has, potentially, a different 
+
+The table below outlines the channels:
+
+| Property         	| Description                                                                                                                                                                                                                                                       	|
+|------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| Dev              	| Unstable bits that are "bleeding edge". Users are nto expected to us this channel often, however it is there for those situations when someone needs a feature or maybe a new bug fix urgently that hasn't been stabilizied yet. Also, used for internal testing. 	|
+| Beta/Pre-release 	| Pre-release stable builds with known issues and/or known feature gaps. We are OK with users installing these for preview purposes.                                                                                                                                	|
+| Release          	| Actual releases. Most users are encouraged to install these.                                                                                                                                                                                                      	|
+Channels also impact the NuGet packages. Based on the version of the package specified, the user may end up getting different channels. This is described in more details in [Nuget Packages section](#nuget-packages).  
+
+# Dependencies
+.NET Core CLI is built on top of CoreFX and CoreCLR and as such its' dependencies set is defined by the platform that those two combine. Some of the install options provide an automatic way to install these dependencies, while other do not. However, it is important to note that the CLI bundle will **not carry those dependencies with it** in any case. This means that for those install options that do not support automatic 
+
+A list of dependencies can be found on [](). 
+
+# Upgrades
+**TODO**
+
+# Layout on disk
```

Comment:
this should be only specific so it is clear how to run dotnet.exe but not specific enough so we have some room to do some changes. Ideally i believe we should only document that ./bin/dotnet.exe is guaranteed to exists and nothing else isn't (+ other stuff we want to promise if there is any)